swap/swap_test.go

```diff
@@ -88,6 +91,15 @@ func init() {
 	testBackend = newTestBackend()
 	// commit the initial "pre-mined" accounts (issuer and beneficiary addresses)
 	testBackend.Commit()
+
+	var err error
```

Comment:
you can omit this line by using `:=` on line `97`

Reply:
fixed